lib/assets/stylesheets/nyulibraries/_breadcrumbs.scss

```diff
@@ -5,7 +5,7 @@
      -moz-border-radius: 0;
           border-radius: 0;
   color: lighten(black, 20%);
-  background-color: $breadcrumb-bg;
+  background-color: $breadcrumbs-bg;
```

Comment:
@Amay22 The bug is actually here: https://github.com/NYULibraries/nyulibraries-assets/blob/master/lib/assets/stylesheets/nyulibraries/_variables.scss#L17

That should say `$breadcrumb-bg` to match the default variables here: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L788

And then you can revert these back to `breadcrumb-bg` singular.